index.js

```diff
@@ -13,6 +13,16 @@ module.exports.pitch = function (request) {
 	if (this.cacheable) this.cacheable();
 
 	var options = loaderUtils.getOptions(this) || {};
+	var context = options.context || this.rootContext || (this.options && this.options.context);
+	var attrs = {};
+	for(var key in options.attrs||{}){
+		var name = loaderUtils.interpolateName(this,options.attrs[key],{
```

Comment:
Can we add simple regex to detect `[here]`? To avoid perf problem?

Reply:
I add a flag and a regexp